src/main/java/uk/gov/hmcts/divorce/common/model/SupportingDocuments.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.divorce.common.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.hmcts.ccd.sdk.api.HasLabel;
+
+@Getter
+@AllArgsConstructor
+public enum SupportingDocuments implements HasLabel {
```

Comment:
could we use singlar class names, ie `SupportingDocument`. And perhaps this class should be `SupportingDocumentType` if it's not the document itself